easybuild/easyconfigs/f/FLEUR/FLEUR-MaXR2.1-intel-2018a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'FLEUR'
+version = 'MaXR2.1'
```

Comment:
Why not using a version like `0.27-MaXR2.1`?

Reply:
well, I guess that depends on whether there will ever be a `0.28` version :)

but you're certainly right about the continuity aspect, will do